db/keyval/redis/bytes_broker_impl.go

```diff
@@ -187,169 +178,87 @@ func (db *BytesConnectionRedis) ListValues(match string) (keyval.BytesKeyValIter
 	return &bytesKeyValIterator{values: kvs}, nil
```

Comment:
Yes, I will implement paging once the Iterator interface is updated (I think this update will impact etcd too)